aten/src/ATen/native/nested/NestedTensorBinaryOps.cpp

```diff
@@ -174,6 +174,13 @@ Tensor NestedTensor_add_Tensor(
       });
 }
 
+Tensor NestedTensor_add_Tensor_meta(
+    const Tensor& self,
+    const Tensor& other,
+    const Scalar& alpha) {
+  return at::empty_like(self);
+}
```

Comment:
Sums now add supports striding ( by calling contiguous) I think empty like will fail unless you do self.contiguous

Reply:
totally agree

Reply:
totally agree